### DISCUSSION TITLE
Figma Variables via URL

### DISCUSSION THREAD

**Discussion by OmarJalilo** [lukasoppermann/design-tokens#313] [Q&A]
Currently we export figma variables manually but we want to use it via URL, we already have configured all the github actions. This current manual configuration works for us. I see that setting is not included inside the URL options. Do you have an ETA for that?

<img width="561" alt="Screenshot 2024-07-29 at 12 59 11 p m" src="https://github.com/user-attachments/assets/f33fbc4b-140f-421a-9f84-e92e75f6c410">

**Comment by lukasoppermann** [comment]
Sorry, no. If it does not work now, we would need someone to send a PR to add this feature.
